abstract.go

```diff
@@ -39,6 +39,8 @@ func scrapeAwsData(config conf) ([]*tagsData, []*cloudwatchData) {
 
 			resources, metrics := scrapeDiscoveryJob(job, config.Discovery.ExportedTagsOnMetrics, clientTag, clientCloudwatch)
 
+			log.Println("Finished scrap!")
```

Comment:
I think we can remove custom logging for now

Reply:
I had added this log to show the end of scrape execution, but I think we can remove it. :smile: